contracts/0.8.9/DepositSecurityModule.sol

```diff
@@ -49,6 +49,14 @@ contract DepositSecurityModule {
     event DepositsUnpaused(uint24 indexed stakingModuleId);
 
     error ErrorStakingModuleIdTooLarge();
+    error ErrorZeroAddress(string field);
```

Comment:
I'd rather remove the Error prefix.

Reply:
e.g `ErrorSignature` even doesn't explain anything. It's better to be `WrongSignature` or `InvalidSignature` or even `ImpossibleToVerifySignature`.